source/clear-linux/guides/maintenance/developer-workstation.rst

```diff
@@ -0,0 +1,108 @@
+.. _developer-workstation:
+
+Developer Workstation
+#####################
+
+Overview
+********
+
+*Developer Workstation* helps you find the :ref:`bundles-about` you need to 
+start your |CL| development project. 
+
+Before continuing, we recommend that you learn how to use :ref:`swupd <swupd-guide>`. Visit our :ref:`swupd-about` page to understand how |CL|
+simplifies software versioning in contrast to other Linux\* distributions. 
+
+Workstation Setup
+=================
+
+This guide helps you understand the minimum bundles required to get started. 
+After installing them, you can add more bundles relevant to your use case. 
+To run any process required for Clear Linux development, you may choose to 
+add the large bundle :ref:`*os-clr-on-clr* <enable-user-space>`. However, 
+given how many packages this bundle contains, you may instead want to deploy 
+a leaner OS with only those bundles relevant to your project. Developer 
+Workstation responds to this need. 
+
+Use Table 1, *Clear Linux Developer Profiles*, to identify the *minimum 
+required bundles* to get started developing based on your role or project. 
+While your role may not neatly fit in one of these categories, consider using Table 1 as a starting point. 
+
+.. list-table:: **Table 1. Clear Linux Developer Profiles**
+   :widths: 20, 20, 20, 20
+   :header-rows: 1
+
+   * - Clear Linux Bundle
+     - *Internet of Things (IoT)* 
+     - *System Administrator*
+     - *Client/Cloud/Web Developer*
+     
+   * - `editors` 
+     - ✓
+     - ✓
+     - ✓
+
+   * - `network-basic`
+     - ✓
+     - ✓
+     - ✓
+
+   * - `openssh-server`
+     - ✓
+     - ✓
+     - ✓
+   
+   * - `webserver-basic`
+     - 
+     - ✓
+     - ✓   
+   
+   * - `application-server`
+     - 
+     - ✓
+     - ✓
+   
+   * - `database-basic`
+     - 
+     - ✓
+     - ✓
+   
+   * - `desktop-autostart`
```

Comment:
This looks good but I would still recommend that any user of linux use a desktop.  We live in a gui world and it does make life easier!

Reply:
Done

Reply:
done